apps/root/src/pages/home/components/portfolio/index.tsx

```diff
@@ -26,14 +39,50 @@ const StyledAssetLogosContainer = styled.div`
 
 export type PortfolioRecord = Record<
   string,
-  { balance: number; price?: number; token: Token; isLoadingPrice: boolean }
+  { balance: BigNumber; balanceUsd?: number; price?: number; token: Token; isLoadingPrice: boolean }
 >;
 
 interface PortfolioProps {
   balances: PortfolioRecord;
+  isLoadingAllBalances: boolean;
 }
 
-const Portfolio = ({ balances }: PortfolioProps) => {
+const PortfolioBodySkeleton = () => {
+  const skeletonRows = Array.from(Array(5).keys());
+  return (
+    <>
+      {skeletonRows.map((i) => (
+        <TableRow key={i}>
+          <TableCell>
+            <Grid container gap={1} direction="column">
+              <Skeleton variant="rounded" width={150} height={24} />
```

Comment:
can't you use the `text` variant? So we don't have to defijne width and height

Reply:
Definitely! Thank you. Font sizes can be adjusted later based on the design